### ISSUE TITLE
 testsuite: TypeError: test_ref() got an unexpected keyword argument 'kind'

### ISSUE THREAD

**Issue by victoredwardocallaghan** [servo/servo#9986]
Hi,

The test suite failed on me today :| servo was built from `commit a862384841cb6e8ff43f627a59b67c4f5189b81b`.

```
[edward@localhost servo]$ ./mach test
Checking files for tidiness...
  Progress: 100% (7718/7718)
Running the WPT lint...
  Progress: 100% (18707/18707)
tidy reported no errors.
Error running mach:

    ['test']

The error occurred in code that was called by the mach command. This is either
a bug in the called code itself or in the way that mach is calling it.

You should consider filing a bug for this issue.

If filing a bug, please include the full output of mach, including this error
message.

The details of the failure are as follows:

TypeError: test_ref() got an unexpected keyword argument 'kind'

  File "/home/edward/servo/python/servo/testing_commands.py", line 156, in test
    Registrar.dispatch("test-%s" % suite, context=self.context, **kwargs)
  File "/home/edward/servo/python/mach/mach/registrar.py", line 115, in dispatch
    return self._run_command_handler(handler, context=context, **kwargs)
  File "/home/edward/servo/python/mach/mach/registrar.py", line 90, in _run_command_handler
    result = fn(**kwargs)
```

**Comment by Ms2ger** [comment]
The
```python
            ("ref", {"kwargs": {"kind": render_mode},
                     "paths": [path.abspath(path.join("tests", "ref"))],
                     "include_arg": "include"}),
```
bit should be removed from `python/servo/testing_commands.py`. Do you want to submit a pull request to fix that?

**Comment by malayaleecoder** [comment]
I would like to work on this. Almost done, just the testing left :)

**Comment by Wafflespeanut** [comment]
(easy part of #9544)

**Comment by victoredwardocallaghan** [comment]
We can probably close this now with fixed in 4b9ee25 right?

**Comment by Wafflespeanut** [comment]
It will be closed once the PR gets merged :)
